rocAL/rocAL/include/pipeline/master_graph.h

```diff
@@ -113,6 +125,16 @@ class MasterGraph
     std::list<std::shared_ptr<Node>> _root_nodes;//!< List of all root nodes (image/video loaders)
     std::list<std::shared_ptr<Node>> _meta_data_nodes;//!< List of nodes where meta data has to be updated after augmentation
     std::map<rocalTensor*, std::shared_ptr<Node>> _tensor_map;//!< key: tensor, value : Parent node
+
```

Comment:
@fiona-gladwin TODO: Can we have the metadata Tensor list to be a list and also store a string which has the name of metadata?

Reply:
@fiona-gladwin What need to do here?